ui/src/app/pages/admin/user/admin-user-common.tsx

```diff
@@ -122,19 +122,28 @@ export const getPublicInstitutionDetails = async (): Promise<
 export const getInitialCreditLimitOptions = (
   initialCreditLimitOverride?: number
 ) => {
+  const STEP1 = 25;
```

Comment:
could make this more structured/repeatable, but I don't think there's much value unless we need to keep going with more groups